handler/core/src/main/java/io/knotx/fragments/handler/action/cb/CircuitBreakerActionFactoryOptions.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (C) 2019 Knot.x Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.knotx.fragments.handler.action.cb;
+
+import io.vertx.circuitbreaker.CircuitBreakerOptions;
+import io.vertx.codegen.annotations.DataObject;
+import io.vertx.core.json.JsonObject;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.apache.commons.lang3.StringUtils;
+
+@DataObject(generateConverter = true)
+public class CircuitBreakerActionFactoryOptions {
+
+  /**
+   * Default value of the fallback on failure property.
+   */
+  private static final boolean DEFAULT_FALLBACK_ON_FAILURE = true;
+
+  private static final String DEFAULT_NODE_LOG_LEVEL = "error";
+
+  private static final CircuitBreakerOptions DEFAULT_CIRCUIT_BREAKER_OPTIONS = initCircuitBreakerOptions(
+      new CircuitBreakerOptions());
+
+  private String circuitBreakerName;
+
+  private CircuitBreakerOptions circuitBreakerOptions = DEFAULT_CIRCUIT_BREAKER_OPTIONS;
+
+  private String logLevel = DEFAULT_NODE_LOG_LEVEL;
```

Comment:
Maybe it should be:
```suggestion
  private String logLevel = ERROR.getLevel();
```
from `io.knotx.fragments.handler.api.actionlog.ActionLogLevel`